beacon-chain/powchain/options.go

```diff
@@ -10,6 +13,9 @@ import (
 	"github.com/prysmaticlabs/prysm/network"
 )
 
+// DefaultRPCHTTPTimeout for HTTP requests via an RPC connection to an execution node.
+const DefaultRPCHTTPTimeout = time.Second * 5
```

Comment:
we need to be careful with this number, some our requests when syncing deposit logs require the execution node to answer with large payloads. These can take some time.

Reply:
half a slot should be sufficient, perhaps. What do you think about 6 seconds @terencechain ?

Reply:
Six is fine with me. Ideally, the different RPC calls should have different timeout constraints but we could add that later